### ISSUE TITLE
Update results view to show real results

### ISSUE THREAD

**Issue by curtislmartin** [Kira-Le/badminton_stroke_classification#87]
First confirm and document what is currently being shown in the results.

Results view must have:
1. Training set results
2. Test set results
3. Filterable by Shot type, Match, Gender, ..... (to be expanded)
4. Show and compare _results_ by each model

Create a separate enabled results view to replace the Project button. 
This will show and compare all of the results between models, and not be gated by the classification flow.

New results view for real inference, showing both models results
